doc/DateTime.md

```diff
@@ -27,6 +27,14 @@ You may specify the format of the date&time, for details see https://momentjs.co
 The default format is `lll` -> `Aug 22, 2020 1:13 PM`
 
 
+### Optional prop
+The prop `dateTimeFormat` specify length of the time.
+(Optional. The default format is `dateTimeFormat="medium` -> `Nov 22, 2022, 1:13 PM`)
```

Comment:
@Aringoaway Forgotten closing quotes for medium, pls fix `dateTimeFormat="medium"`